### ISSUE TITLE
CMake 3.15.1 crashes when building FairRoot v18.0.7

### ISSUE THREAD

**Issue by fuhlig1** [FairRootGroup/FairRoot#905]
When using CMake 3.15.1 with FairRoot v18.0.7, CMake stops with the following error message

```
CMake Error at MbsAPI/CMakeLists.txt:19 (CHECK_SYMBOL_EXISTS):
  Unknown CMake command "CHECK_SYMBOL_EXISTS".
```

When using older CMake versions like 3.11.1 (part of FairSoft may18) or 3.13.4 (part of FairSoft jun19) the problem does not show up.

To reproduce the error use CMake 3.15.1 together with FairRoot v18.0.7

The expected behavior is that CMake runs without problems when using CMake 3.15.1 

The problem was seen with macosx 10.12 but I expect that the problem also occurs for other versions of macosx and also on Linux. 

The error do not show up with FairRoot v18.2.0 and FairRoot dev.

Looking into the problem one sees that the unknown CMake command "CHECK_SYMBOL_EXISTS" is defined in the file CheckSymbolExists.cmake. This file has to be loaded before usage by the command "include(CheckSymbolExists)" which is missing in FairRoot v18.0.7. In FairRoot v18.2.0 and FairRoot dev the include statement is in the main CMakeLists.txt. Most probably in older CMake versions the file is included implicitly by some other include which changed with the latest CMake version.

The fix is rather easy. Either add the missing include statement in the main CMakeLists.txt as done in FairRoot v18.2.0 or add it in the specific CMakeLists.txt which has the problem.

**Comment by dennisklein** [comment]
See latest comment in alisw/alidist#1614.

**Comment by dennisklein** [comment]
Fixed in `v18.0.8`
